portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -1083,6 +1084,7 @@ definition {
 			rowEntry = "Created Form Name");
 	}
 
+	@ignore = "true"
```

Comment:
@alinedoleron, the SF issue is occurring because `@ignore` should be added following the alphabetic order, e.g. it should be placed after `@description`.